.github/workflows/create_tag.yml

```
@@ -44,5 +44,8 @@ jobs:
     needs: create_tag
     if: needs.create_tag.outputs.new_tag != '' && needs.create_tag.outputs.new_tag != 'undefined'
     uses: ./.github/workflows/release.yml
+    permissions:
+      content: write
```

actionlint (1.7.9)

unknown permission scope "content". all available permission scopes are "actions", "attestations", "checks", "contents", "deployments", "discussions", "id-token", "issues", "models", "packages", "pages", "pull-requests", "repository-projects", "security-events", "statuses"

(permissions)